### PR TITLE
Added stmyec_wcm_2022 study

### PR DESCRIPTION
**Pull request by alexsigaras**
# What?
Fix # .

Cancer studies updated in this pull request:
- stmyec_wcm_2022
- .

# checks
For all pull requests:
- [x] Passes validation

For a new study (in addition to above):
- [x] Does study name and study ID follow our convention? e.g. Tumor_Type (Institue, Journal Year); brca_mskcc_2015
- [x] is study meta data complete? e.g. pmid, group of PUBLIC
- [x] were all samples profiled with WES/WGS? If not, is gene panel file curated?
- [ ] are oncotree codes of all samples curated; Cancer Type and Cancer Type Detailed needs to be added in addition to Oncotree Code
- [x] clinical sample and patient data with meta files
- [x] mutations data with meta files
- [x] MAF is based on hg19
- [ ] MAF with 2 isoforms: uniprot and mskcc
- [x] CNA data with meta files
- [x] CNA segment data with meta files
- [ ] Expression data including z-scores with meta files
- [x] Case-lists for all profiles.
- [ ] Manual checking (Niki or JJ): Triage or private Portal link here

### CODE REVIEW

File: public/stmyec_wcm_2022/meta_study.txt [hunk 1]
```diff
@@ -0,0 +1,8 @@
+type_of_cancer: stmyec
+cancer_study_identifier: stmyec_wcm_2022
+name: Myoepithelial Carcinomas of the Soft Tissue (CSH Molecular Case Studies, 2022)
```
Comment (ritikakundra) [inline public/stmyec_wcm_2022/meta_study.txt:3]: Can you change the name to  Myoepithelial Carcinomas of the Soft Tissue (WCM, CSH Molecular Case Studies 2022)

File: public/stmyec_wcm_2022/meta_study.txt [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+description: Whole Exome Sequencing results (mutation and copy number data) for Weill Cornell Medicine Myoepithelial Carcinomas of the Soft Tissue cohort
```
Comment (ritikakundra) [inline public/stmyec_wcm_2022/meta_study.txt:4]: Are these with matched normals?

**Review by ritikakundra — CHANGES_REQUESTED** [review]
- is it possible to replace '.' from sample IDs to '-'? 
- Would it be possible to run https://github.com/cBioPortal/datahub-study-curation-tools/tree/master/scripts on this study?

File: public/stmyec_wcm_2022/meta_study.txt (hunk 1 above)
Comment (alexsigaras) [reply public/stmyec_wcm_2022/meta_study.txt:3]: Thanks. Will update shortly.

File: public/stmyec_wcm_2022/meta_study.txt (hunk 2 above)
Comment (alexsigaras) [reply public/stmyec_wcm_2022/meta_study.txt:4]: Yes. I'll update in the description as well.

**Comment by alexsigaras** [comment]
> * is it possible to replace '.' from sample IDs to '-'?
> * Would it be possible to run https://github.com/cBioPortal/datahub-study-curation-tools/tree/master/scripts on this study?

Yes and Yes and will be pushing the updated cohort shortly.
Please note that this doesn't run properly on python3 and have created a PR on the tool that solves the problem
https://github.com/cBioPortal/datahub-study-curation-tools/pull/52

File: public/stmyec_wcm_2022/meta_study.txt (hunk 1 above)
Comment (alexsigaras) [reply public/stmyec_wcm_2022/meta_study.txt]: @ritikakundra : Name changed

File: public/stmyec_wcm_2022/meta_study.txt (hunk 2 above)
Comment (alexsigaras) [reply public/stmyec_wcm_2022/meta_study.txt]: @ritikakundra : Description changed to reflect changes

**Review by ritikakundra — APPROVED** [review]

**Pull request merged**